arc-mlir/src/tests/arc-to-rust/loops.mlir.rust-tests

```diff
@@ -1,14 +1,15 @@
 #[cfg(test)]
 mod tests {
+  use super::*;
   use crate::arctorustloops::*;
   use hexf::*;
   #[rewrite(main)]
   #[test]
   fn it_works() {
-    assert_eq!(call!(a_while_loop(0, 5, 3)), 13);
-    assert_eq!(call!(a_while_loop_with_a_break_in_before(0, 5, 1)), 2);
-    assert_eq!(call!(a_while_loop_with_a_break_in_after(0, 10, 3)), 4);
-    assert_eq!(call!(a_while_loop_with_a_return_in_before(0, 5, 1)), 2);
-    assert_eq!(call!(a_while_loop_with_a_return_in_after(0, 10, 3)), 4);
+    // assert_eq!(call!(a_while_loop(0, 5, 3)), 13);
```

Comment:
This looks fishy, if none of the tests work, why isn't the tests XFAIL?

Reply:
With `call!(...)` it's not possible to call functions defined in Arc-Lang from Rust, only the other way around 🤔. The stuff inside `loops.mlir` is still compiled and checked.

Reply:
Ah, makes sense, but in that case why doesn't, for example, the `arc-cmpf` and `arc-cmpi` tests fail in the same way?

Reply:
Or wait, I tested it now and it compiles. It's possible to `call!(...)` but only on functions defined in arc-lang

Reply:
I reverted the change 👍

Reply:
I should apparently trust my nose :)